abi/abi.go

```diff
@@ -702,3 +702,44 @@ func (c *CertTable) Proto() *pb.CertificateChain {
 		FirmwareCert: firmware,
 	}
 }
+
+// See assembly implementations in cpuid_*.s
```

Comment:
Maybe a bit more description on what this function does? It may not be straightforward to understand it from the assembly code.

Reply:
Comment added.